rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -433,17 +434,17 @@ func (r *Layer2Relayer) ProcessCommittedBatches() {
 		r.metrics.rollupL2RelayerProcessCommittedBatchesFinalizedTotal.Inc()
 
 		// Check batch status before send `finalizeBatchWithProof` tx.
-		//batchStatus, err := r.getBatchStatusByIndex(batch.Index)
-		//if err != nil {
-		//	r.metrics.rollupL2ChainMonitorLatestFailedCall.Inc()
-		//	log.Warn("failed to get batch status, please check chain_monitor api server", "batch_index", batch.Index, "err", err)
-		//	return
-		//}
-		//if !batchStatus {
-		//	r.metrics.rollupL2ChainMonitorLatestFailedBatchStatus.Inc()
-		//	log.Error("the batch status is not right, stop finalize batch and check the reason", "batch_index", batch.Index)
-		//	return
-		//}
+		batchStatus, err := r.getBatchStatusByIndex(batch.Index)
```

Comment:
I think the checker `if !r.cfg.ChainMonitor.EnableChainMonitor {` put here will be more suitable

Reply:
fixed.